docs/IR.md

```diff
@@ -65,9 +65,12 @@ Each computation node consists of a name, the identifier of the operator to be i
 
 Edges in the computation graph are established by outputs of one node being referenced by name in the inputs of a subsequent node.
 
+Names of tensor values flowing through the graph are unique - a particular value can be either graph input or an output of a single node. Reuse or reassingment of values is not allowed.
```

Comment:
Suggested changes:
(1) Change "a particular value can be either" to "a particular name can reference either a"
(2) Change "reassignment of values" to "reassignment of values to names"